composer.json

```diff
@@ -102,7 +102,8 @@
         "symfony/phpunit-bridge": "^5.0",
         "snc/redis-bundle": "^3.1",
         "predis/predis": "^1.1.1",
-        "franzl/studio": "*"
+        "franzl/studio": "*",
+        "captainhook/plugin-composer": "^4.0"
```

Comment:
Should be upgraded to `^5.1`

Reply:
Done in 880a347b0648d41b7ca38d81129a124e7343e3f9